docs/layers.md

```diff
@@ -434,21 +842,21 @@ Railway values in this layer include: `rail`, `tram`, `light_rail`, `narrow_gaug
 
 ![image](images/mapzen-vector-tile-docs-roads-airport.png)
 
-**Airport** aeroways with `kind` values of `runway` show up at zoom 9, with `taxiway` at zoom 11+.
+**Airport** aeroways with `subkind` values of `runway` show up at zoom 9, with `taxiway` at zoom 11+.
```

Comment:
We've talked about this on and off. I've finally added an issue to track it here: https://github.com/mapzen/vector-datasource/issues/719. (It's easier to accomplish now we have the YAML lookups!)

The old value in the docs for `kind` was actually incorrect. Updated it towards the intended state for this release rather than having intermediate throw away text.